code/modules/clothing/shoes/colour.dm

```diff
@@ -96,11 +96,8 @@
 
 /obj/item/clothing/shoes/sneakers/orange/attackby(obj/H, loc, params)
 	..()
-	if ((istype(H, /obj/item/weapon/restraints/handcuffs) && !( src.chained )))
-		//H = null
-		if (src.icon_state != "orange") return
-		if(istype(H, /obj/item/weapon/restraints/handcuffs/cable))
-			return 0
+	// Note: not using istype here because we want to ignore all subtypes
+	if (H.type == /obj/item/weapon/restraints/handcuffs && !chained && icon_state == "orange")

```

Comment:
The `icon_state == "orange"` bit is just because seekers can have their sprite changed by things like the washing machine. It's actually a purely aesthetic check, and could be removed with no impact to actual behavior.